src/main/java/com/bart/scorebetlive442/model/json/datajson/03_createMatch.json

```diff
@@ -0,0 +1,21 @@
+{
+  "teamHome": {
+    "id": 1,
```

Comment:
rozumiem że to na razie trzymasz w celu sprawdzenia cascady, docelowo w create nie powinniśmy podawać całości (chyba że dedykowanym endpointem).